src/python/WMCore/MicroService/MSOutput/MSOutput.py

```diff
@@ -542,8 +543,9 @@ def msOutputConsumer(self):
                     msg = "%s General error from pipeline. Err: %s. " % (pipeLineName, str(ex))
                     msg += "Will retry again in the next cycle."
                     self.logger.exception(msg)
-                    break
-                wfCounters += 1
+                    workflowname = docOut.get("_id", "none")
```

Comment:
This string `"none"` might be confusing at some point. I would suggest to either set it to the `None` object, or to set it to an empty string `""`.

Reply:
i see, ok